### DISCUSSION TITLE
Unable to get Realtek Audio Control to install

### DISCUSSION THREAD

**Discussion by kingmustard** [pal1000/Realtek-UAD-generic#88] [Q&A]
I have Unofficial-Realtek-UAD-generic-6.0.9215.1 installed.

When I try to install [Realtek Audio Control](www.microsoft.com/en-gb/p/realtek-audio-control/9p2b8mcsvpln) from the Microsoft Store, I simply get "Error".

When I try to install from RealtekAudioControl_1.27.254-offline (Microsoft.VCLibs.140.00_14.0.30035.0_x64__8wekyb3d8bbwe.appx), it asks if I want to install Microsoft Visual C++ 2015 UWP Runtime Package.

If I click Install, installation fails and I get:

> App installation failed with error message: error 0x80073D02: Unable to install because the following apps need to be closed Microsoft.WindowsStore_12107.1001.15.0_x64__8wekyb3d8bbwe
Microsoft.YourPhone_1.21072.160.0_x64__8wekyb3d8bbwe
MicrosoftWindows.Client.CBS_321.13302.10.3_x64__cw5n1h2txyewy windows.immersivecontrolpanel_10.0.6.1000_neutral_neutral_cw5n1h2txyewy.
(0x80073d02)

Any ideas?

**Comment by pal1000** [comment]
A logout / login cycle or restart may solve the problem. Some store apps appear unresponsive or busy.

**Reply by kingmustard** [reply]
> A logout / login cycle or restart may solve the problem. Some store apps appear unresponsive or busy.

Unfortunately, a restart didn't change anything ☹️
